src/components/PersonLink.tsx

```diff
@@ -0,0 +1,141 @@
+import { Link, useLocation } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { GetPersonDates } from '../services/getPersonDates';
```

Comment:
Ensure that the `GetPersonDates` function is correctly imported and defined. If it is supposed to return a slug or identifier for the person's parent, verify its implementation.